startHadoop.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+/opt/hadoop-3.4.0/bin/hdfs namenode -format
```

Comment:
I would not have this in here as a default

Reply:
I will add an env flag. Since I was using tmp directories this was messing me up between ec2 restarts. namenode wasn't starting.